### DISCUSSION TITLE
Using bridge packages outside of Symfony

### DISCUSSION THREAD

**Discussion by msimkunas** [symfony/symfony#58019] [Q&A]
Symfony has bridge and bundle packages. Bundle packages are intended for use with Symfony apps whereas bridge packages seem to be more generic and only extend other libraries.

Am I correct in assuming that bridge packages such as monolog-bridge can be used outside Symfony? Are they designed to be able to be used without the Symfony framework?

**Comment by MatTheCat** [comment]
Indeed, bridges’ goal is to integrate libraries with Symfony **components**. Therefore, they don’t require the framework.

**Reply by msimkunas** [reply]
Is there a docs page or any other resource that describes the difference between Symfony bridges and bundles in more detail?
